src/fmu/dataio/_model/root.py

```diff
@@ -72,6 +74,42 @@ class CaseMetadata(MetadataBase):
     this data object. See :class:`Access`."""
 
 
+class IterationMetadata(MetadataBase):
+    """The FMU metadata model for an FMU Iteration.
+
+    An object representing a single Iteration of a specific case.
+    """
+
+    class_: Literal[FMUClass.iteration] = Field(alias="class", title="metadata_class")
+    """The class of this metadata object. In this case, always an FMU iteration."""
+
+    fmu: FMUIteration
+    """The ``fmu`` block contains all attributes specific to FMU.
+    See :class:`FMU`."""
+
+    access: SsdlAccess
```

Comment:
Is the `Access` model sufficient for these realization/iteration objects? The `SsdlAccess` will in addition add the `ssdl` block in the metadata, and these objects will not be shared with the datalake I guess?

Reply:
Good point, think `Access` should be enough (as it is for Case objects).
Changed now.